client/src/components/DropdownMenu.js

```diff
@@ -0,0 +1,44 @@
+import "./DropdownMenu.scss";
+import classNames from "classnames";
+
+export class DropdownMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+    };
+  }
+  render() {
+    const {
+      dropdownContent,
+      button_class_name,
+      button_description,
+      IconComponent,
+      icon_color,
+    } = this.props;
+    const { isOpen } = this.state;
+
+    return (
+      <div className="dropdown">
+        <button
+          className={button_class_name}
+          onClick={() => this.setState({ isOpen: !isOpen })}
+        >
+          <IconComponent
+            title={button_description}
+            color={icon_color}
+            alternate_color={false}
+          />
+        </button>
```

Comment:
Gonna need to double check this with a screen reader, I think it will read out that it's on a button but won't read out the title of the icon inside it without the user drilling down. Guessing this is a mistake made in other parts of the code base though, only just occurred to me (oops, sure it's a mistake I've also made). Well, test if it seems properly usable at least, I could be wrong about what the screen reader will do.

Reply:
Just tried it out on Mac voiceover, it works as expected (although I got rid of icon anyways)